src/renderer/components/deposit/add/Deposit.helper.ts

```diff
@@ -45,8 +47,9 @@ export const maxRuneAmountToDeposit = ({
       .toFixed(0, BigNumber.ROUND_DOWN),
     THORCHAIN_DECIMAL
   )
-
-  return maxRuneAmount.amount().isGreaterThan(runeBalance.amount()) ? runeBalance : maxRuneAmount
+  return maxRuneAmount.amount().isGreaterThan(runeBalance.amount())
+    ? runeBalance.minus(thorchainFees.inFee)
+    : maxRuneAmount
```

Comment:
In case `maxRuneAmount` === `runeBalance`, fees has to be deducted, too.


Hint: Use helper function included in `BaseAmount` 
```
maxRuneAmount.gt(runeBalance)
```

Reply:
You probably meant `maxRuneAmount.gte(runeBalance)`?

Reply:
Yep.